package.json

```diff
@@ -17,8 +17,16 @@
     "build:umd": "cross-env BABEL_ENV=commonjs webpack",
     "build:umd:min": "cross-env BABEL_ENV=commonjs NODE_ENV=production webpack",
     "build:watch": "npm run build:commonjs -- -watch",
+    "precommit": "lint-staged",
     "prepublish": "npm run clean && npm run build"
   },
+  "lint-staged": {
```

Comment:
I love Prettier, and using lint-staged here is super handy for contributors.  But be aware that this recipe can be very surprising if someone has used `git add -p`: any file that gets reformatted will become part of the commit, even if it was only partially staged.  See https://github.com/okonet/lint-staged/pull/75 for details.